src/Form/Project/EtudeType.php

```diff
@@ -31,11 +33,28 @@ class EtudeType extends AbstractType
 {
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('knownProspect', CheckboxType::class, [
-            'label' => 'suivi.etude_form.client_bdd',
-            'translation_domain' => 'project',
-            'required' => false,
-        ])
+        $builder
+            ->add('boutonCcaActive', CheckboxType::class, [
```

Comment:
```suggestion
            ->add('ccaActive', CheckboxType::class, [
```
Tous les champs des formulaires sont toujours nommés du même nom que la propriété de l'entité. 
Si dans ce cas précis ce n'est pas possible, décrire pourquoi avec un commentaire.

Reply:
C’était nécessaire ;)
L’explication est ajoutée dans le prochain push !

Reply:
Merci pour le commentaire, mais je ne comprends pas pourquoi est ce que le nom influe sur l'affichage du bouton.

Reply:
J’avoue que ça date un peu :|
Quand on met le même nom que la propriété le champ dans le formulaire ne s’affiche même pas. Je pense que ça a à voir avec le code script à la fin du fichier qui cache ou affiche les éléments du DOM… Je ne sais pas vraiment pourquoi ça ne marche pas au fond ^^